From 72e10733f48bf1426fe62eb430895f7c7d5acf95 Mon Sep 17 00:00:00 2001
From: Johannes Kliemann <kliemann@componolit.com>
Date: Wed, 8 May 2019 10:30:04 +0200
Subject: [PATCH] binutils support removed for GCC 2.x mangling schemes with
 1910070 add missing defines to allow compilation on Debian Buster (binutils
 2.31)

---
 support/bfdgen.c | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/support/bfdgen.c b/support/bfdgen.c
index 30ad746..ca9f88f 100644
--- a/support/bfdgen.c
+++ b/support/bfdgen.c
@@ -86,6 +86,30 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.  */
 # define HAS_LOAD_PAGE 0
 #endif
 
+#ifndef SEC_HAS_GOT_REF
+# define SEC_HAS_GOT_REF 0x800
+#endif
+
+#ifndef DMGL_GNU
+# define DMGL_GNU (1 << 9)
+#endif
+
+#ifndef DMGL_LUCID
+# define DMGL_LUCID (1 << 10)
+#endif
+
+#ifndef DMGL_ARM
+# define DMGL_ARM (1 << 11)
+#endif
+
+#ifndef DMGL_HP
+# define DMGL_HP (1 << 12)
+#endif
+
+#ifndef DMGL_EDG
+# define DMGL_EDG (1 << 13)
+#endif
+
 int main(void)
 {
   printf("--  Generated by bfdgen.c from system bfd.h\n");
-- 
2.11.0

